src/components/activity/ActivityDirectivesTable.svelte

```diff
@@ -129,11 +137,22 @@
   function getRowId(activityDirective: ActivityDirective): ActivityDirectiveId {
     return activityDirective.id;
   }
+
+  function scrollToActivityInTimeline() {
+    const directiveId =
+      (bulkSelectedActivityDirectiveIds.length > 0 && bulkSelectedActivityDirectiveIds[0]) ||
+      selectedActivityDirectiveId;
+    const directive = activityDirectives.find(item => item.id === directiveId) ?? null;
+    if (directive?.start_time_ms) {
+      dispatch('scrollToTime', directive.start_time_ms);
+    }
+  }
 </script>
 
 <BulkActionDataGrid
   bind:dataGrid
   bind:selectedItemId={selectedActivityDirectiveId}
+  bind:selectedItemIds={bulkSelectedActivityDirectiveIds}
```

Comment:
@duranb after looking more at this code, had the ActivityDirectivesTable see the array of selectedItems from the BulkActionDataGrid so I can do things specific to this table with the selection, like manage the context menu items specific to Activity Directives.